base/src/main/java/org/apache/cassandra/auth/LDAPAuthenticator.java

```diff
@@ -157,11 +159,10 @@ public AuthenticatedUser authenticate(String username, String password) throws A
                     cacheDelegate.invalidate(user);
                     cacheDelegate.get(user);
                 }
-
-
+                
                 if (cachedUser.getLdapDN() != null && systemAuthRoles.roleMissing(cachedUser.getLdapDN()))
                 {
-                    systemAuthRoles.createRole(cachedUser.getLdapDN(), false);
+                    systemAuthRoles.createRole(cachedUser.getLdapDN(), false, Optional.ofNullable(properties.getProperty(DEFAULT_ROLE_MEMBERSHIP, null)));
```

Comment:
What's the value of passing there Optional which value might be null? Just pass there null already, no?